### PR TITLE
Clarify what's new and test of concatenating anonymous dimensions.

### PR DESCRIPTION
**Pull request by lbdreyer**
Patrick and I tripped over this before I left on holiday, so I thought this clarification was needed.

### CODE REVIEW

File: docs/iris/src/whatsnew/contributions_1.10/newfeature_2015-Jan-21_allow-concat-of-cubes-with-anon-dims.txt [hunk 1]
```diff
@@ -1 +1 @@
-* Cubes with anonymous dimensions can now be concatenated.
+* Cubes with anonymous dimensions can now be concatenated. This can only occur along a dimension that is not anonymous.
```
Comment (pelson) [inline docs/iris/src/whatsnew/contributions_1.10/newfeature_2015-Jan-21_allow-concat-of-cubes-with-anon-dims.txt]: Really this comment should be part of the scope of the concatenation method(s) as well.

**Comment by lbdreyer** [comment]
I have added a comment to the concatenate methods as per @pelson comment

(The build is just failing due to a time out error so this will need to restarted)

**Comment by pelson** [comment]
Thanks @lbdreyer

**Comment by lbdreyer** [comment]
And thank you @pelson for merging it :smile:

**Pull request merged**